lib/bolt/pal/yaml_plan/step/message.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Bolt
+  class PAL
+    class YamlPlan
+      class Step
+        class Message < Step
+          def self.allowed_keys
+            Set['message']
```

Comment:
I think we should allow `name` and `description` here just for consistency. They don't do _much_, but it seems simpler to accept them and let them do nothing than to make this step the exception.